packages/ui/src/coreModules/form/components/FormModal/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import { Modal } from 'semantic-ui-react'
```

Comment:
semantic is an external package so new newline between this import and the prop-types (we add newline, mostly, between external and internal imports)